src/nvim/bytes.c

```diff
@@ -0,0 +1,21 @@
+// Low-level functions with minimal dependencies, side-effects, etc.
+
+#include <assert.h>
+#include <inttypes.h>
```

Comment:
Fixed-size integer types live in stdint. inttypes contains printf format specifiers and some imax function declarations. I am not sure what is the style here though: C99 says that inttypes should also include stdint, so it is safe to use it here.